client/checkout/woopay/express-button/woopay-express-checkout-button.js

```diff
@@ -71,7 +71,8 @@ export const WoopayExpressCheckoutButton = ( {
 		borderRadius = buttonAttributes.borderRadius || borderRadius;
```

Comment:
Found a small issue related to the border-radius which I think we could fix here by using `??` instead of `||`.

```suggestion
		borderRadius = buttonAttributes.borderRadius ?? borderRadius;
```

The issue appears when I refresh the page with the border radius set to `0`. It uses the default border-radius from the CSS, because it tries to apply an invalid inline border-radius `undefined + 'px'`.

https://github.com/user-attachments/assets/8bf6c6d2-69f0-47ff-a280-fedf2fed80db

Reply:
Good find! Thank you!